src/Debugger/Impl/rtvs/R/grid.R

```diff
@@ -30,7 +26,11 @@ grid.data <- function(x, rows, cols) {
   onedim <- ((length(rows) == 1) || (length(cols) == 1));
   if (onedim || is.matrix(x)) {
     if (onedim) {
-      data <- grid.format(x[rows, cols]);
+      if(is.list(x) || is.vector(x) || is.ts(x)) {
```

Comment:
This excludes vectors that have named elements. A better way to check would be `is(x, 'vector')`. Because lists are also vectors, this would make `is.list` check redundant, as well.